doc/statistics/univariate_statistics.qbk

```diff
@@ -121,12 +212,17 @@ In general, you can store your data in an Eigen array, and Armadillo vector, `st
 These routines are usable in float, double, long double, and Boost.Multiprecision precision, as well as their complex extensions whenever the computation is well-defined.
 For certain operations (total variation, for example) integer inputs are supported.
 
+/Nota bene/: The default execution policy for every function is std::execution::seq.
+The use of parallel execution policies (std::execution::par or par_unseq) generally doubles the error introduced in the computation, but is still minimal.
+
 [heading Mean]
 
     std::vector<double> v{1,2,3,4,5};
     double mu = boost::math::statistics::mean(v.cbegin(), v.cend());
     // Alternative syntax if you want to use entire container:
     mu = boost::math::statistics::mean(v);
+    // Alternative syntax if you want to use parallel execution:
+    mu = boost::math::statistics::mean(std::execution::par, v);
 
 The implementation follows [@https://doi.org/10.1137/1.9780898718027 Higham 1.6a].
 The data is not modified and must be forward iterable.
```

Comment:
I think it now has to be random access, not just forward iterable.

Reply:
Forward iterators are all that is required; parallel tests with `std::forward_list` still pass. The only changes I made to your battery of unit tests was to adjust the tolerances.

Reply:
@mborland : Doesn't the complexity of the algorithm increase for `std::forward_list`?

Reply:
Yes, it will.